### ISSUE TITLE
document git-merge-changelog for devs

### ISSUE THREAD

**Issue by mr-c** [dib-lab/khmer#614]
An optional aid to merging

```
mcrusoe@athyra:~/khmer$ apt-cache show git-merge-changelog 
[...]
Description: git merge driver for GNU ChangeLog files
 ChangeLog files *always* foul up most version control systems in
 their default configuration.
 .
 git-merge-changelog is a tool from gnulib designed to help with this
 for the case of GNU-style ChangeLogs; it can be used with at least
 git, bzr, and hg.
```

**Comment by standage** [comment]
Is this package still available? I'm not seeing it on 12.04 or 14.04. Not that I ended up needing/using it, but we might want to mark this advice as deprecated.

**Comment by mr-c** [comment]
Yep!

For Ubuntu: 14.10+ http://packages.ubuntu.com/search?keywords=git-merge-changelog
For Debian: it will be part of the upcoming 'Jessie' stable release. https://packages.debian.org/search?keywords=git-merge-changelog

There's a backport in the GED Lab PPA: https://launchpad.net/~misterc/+archive/ubuntu/gedlab
(the source package name is `gnulib`)

**Comment by mr-c** [comment]
```
sudo apt-add-repository ppa:misterc/gedlab
sudo apt-get update
sudo apt-get install git-merge-changelog
```

**Comment by elmbeech** [comment]
i will work on that

**Comment by mr-c** [comment]
Fixed in #1222
